cidr-filtering-cni

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash -e
+
+if [[ ${DEBUG} -gt 0 ]]; then set -x; fi
+
+exec 3>&1
+exec &>> /tmp/bash-cni-plugin.log
+
+stdin=`cat /dev/stdin`
+tmpfile="/tmp/bash-cni-plugin.log" # todo: remove
```

Comment:
hm, not exactly sure - a log file could probably come in handy. 

Feel free to remove it, then re-add a better "version" in a follow-up PR.